Core/HLE/proAdhoc.h

```diff
@@ -32,6 +32,7 @@
 #include "Core/HLE/sceKernel.h"
 #include "Core/HLE/sceKernelMutex.h"
 #include "Core/HLE/sceUtility.h"
+#include "UI/ChatScreen.h"
```

Comment:
GetChatCount() is the function to get the size of ChatLog vector right so i can make a logic to update the chat screen (call updateChat() function) and get data from chat data on proAdhoc.cpp , or it used to get the new chat counter from other user to update the choice with value display in top of emuscreen?